### ISSUE TITLE
Slider doesn't show keyboard focus

### ISSUE THREAD

**Issue by hjohn** [mkpaz/atlantafx#124]
Run the AtlantaFX Sampler app.

Go to the slider section.  Click on a slider.  Notice that it doesn't indicate it has the focus.  Keyboard presses (left/right/up/down/home/end) however do work.

**Comment by mkpaz** [comment]
Focus and accessibility are both pain points of this theme. It's rather hard to implement, because of the lack of CSS support and a sub-optimal control scene graph. It's very easy for web, because of the support for :before, :after, box-shadow, etc. I spent days playing with different styles and approaches, but some components still don't support it.
